weave/tests/trace/test_client_trace.py

```diff
@@ -1708,7 +1708,7 @@ def _wrapped_fn(*args):
     [
         map_simple,
         map_with_threads_no_executor,
-        map_with_thread_executor,
+        # map_with_thread_executor,  # <-- Flakes in CI
```

Comment:
is this new or old?

Reply:
old, just commenting out again because it seems to flake a lot